src/components/SdkSection/SdkSection.module.css

```diff
@@ -0,0 +1,42 @@
+.sdkSection {
+    display: flex;
+    gap: 24px;
+
+    @media (max-width: 768px) { /* max-width: ${theme.layout.tablet} */
+        flex-direction: column;
+    }
+}
+
+.sdkSectionTitle {
+    display: flex;
+    align-items: center;
+    gap: 12px;
+    margin-bottom: 8px;
```

Comment:
Can you please use tokens from the theme for spacing? (ideally everywhere)

Reply:
Same as https://github.com/apify/apify-docs/pull/995#discussion_r1595558175 
I would also add that in the case of the gap, we don't have a theme.spacing.space12. There's only 8 and 16

Reply:
Yeah, I'd probably prefer to move the styling to the styled-component instead of the modules here. There you can simply use our design system tokens - it's worth to do it this way for the long-term consistency.